### DISCUSSION TITLE
How can I define two applications use the same PVC

### DISCUSSION THREAD

**Discussion by shikingram** [kubevela/kubevela#5051] [Q&A] [answered]
Hi there,
As the title says, if I have two Applications, and they use the same PVC, how can i manage them? Who is responsible for his lifecycle.

forexample:

Application A definition a pvcA,then A created pvcA
Application B referenced pvcA

if I deleted Application A ,maybe pvcA also deleted.
Does Application B affected?

**Comment by shikingram** [comment]
Are there some standerd practice for dealing with this scenario？

**Comment by wonderflow** [comment, marked as answer]
There's a shard resource policy for this work https://kubevela.net/docs/end-user/policies/shared-resource

**Reply by wonderflow** [reply]
shared resource is now only supporting Component, so you may need to defince the PVC as a component for sharing.

**Reply by shikingram** [reply]
thanks, I have got this.
